Documentation/ApiOverview/Events/Hooks/Index.rst

```diff
@@ -20,50 +25,43 @@ implemented the Hook Concept itself doesn't prevent it from being used in any wa
 Using hooks
 ===========
 
-The two lines of code below are an example of how a hook is used for
+The two lines of code below are an example of how a hook can be used for
 clear-cache post-processing. The objective of this could be to perform
 additional actions whenever the cache is cleared for a specific page:
 
-.. code-block:: php
-   :caption: EXT:site_package/ext_localconf.php
-
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tcemain.php']['clearCachePostProc'][] =
-      \MyVendor\Package\Hook\DataHandlerHook::class . '->postProcessClearCache';
+..  literalinclude:: _ext_localconf_addhook.php
+    :language: php
+    :caption: EXT:site_package/ext_localconf.php
 
-This registers the class/method name to a hook inside of
-:php:`\TYPO3\CMS\Core\DataHandling\DataHandler`. The hook will call the user
-function after the clear-cache command has been executed. The user function
+This hook registers the class/method name to a hook inside of
+:php:`\TYPO3\CMS\Core\DataHandling\DataHandler`. The hook calls the user
+function after the `clear-cache` command has been executed. The user function
```

Comment:
Hmm, can we write "after the cache has been cleared" to avoid confusion with commands?